### PR TITLE
fix(nxtp): improve nxtp timeout handling

### PR DESCRIPTION
**Pull request by mathiasmoeller**

### CODE REVIEW

**Comment by mathiasmoeller** [comment]
Rebased on #22

File: src/Lifi.unit.spec.ts [hunk 1]
```diff
@@ -46,7 +47,7 @@ describe('LIFI SDK', () => {
         const request = getRoutesRequest({ fromChainId: 'xxx' })
 
         await expect(Lifi.getRoutes(request)).rejects.toThrow(
-          'SDK Validation: Invalid Routs Request'
+          'Invalid Routs Request'
```
Comment (Addminus) [inline src/Lifi.unit.spec.ts:50]: typo: Routes

File: src/Lifi.unit.spec.ts [hunk 2]
```diff
@@ -55,7 +56,7 @@ describe('LIFI SDK', () => {
         const request = getRoutesRequest({ fromAmount: 10000000000000 })
 
         await expect(Lifi.getRoutes(request)).rejects.toThrow(
-          'SDK Validation: Invalid Routs Request'
+          'Invalid Routs Request'
```
Comment (Addminus) [inline src/Lifi.unit.spec.ts:59]: typo: Routes

File: src/Lifi.unit.spec.ts [hunk 3]
```diff
@@ -64,7 +65,7 @@ describe('LIFI SDK', () => {
         const request = getRoutesRequest({ fromTokenAddress: 1234 })
 
         await expect(Lifi.getRoutes(request)).rejects.toThrow(
-          'SDK Validation: Invalid Routs Request'
+          'Invalid Routs Request'
```
Comment (Addminus) [inline src/Lifi.unit.spec.ts:68]: typo: Routes

File: src/Lifi.unit.spec.ts [hunk 4]
```diff
@@ -73,7 +74,7 @@ describe('LIFI SDK', () => {
         const request = getRoutesRequest({ toChainId: 'xxx' })
 
         await expect(Lifi.getRoutes(request)).rejects.toThrow(
-          'SDK Validation: Invalid Routs Request'
+          'Invalid Routs Request'
```
Comment (Addminus) [inline src/Lifi.unit.spec.ts:77]: typo: Routes

File: src/Lifi.unit.spec.ts [hunk 5]
```diff
@@ -82,7 +83,7 @@ describe('LIFI SDK', () => {
         const request = getRoutesRequest({ toTokenAddress: '' })
 
         await expect(Lifi.getRoutes(request)).rejects.toThrow(
-          'SDK Validation: Invalid Routs Request'
+          'Invalid Routs Request'
```
Comment (Addminus) [inline src/Lifi.unit.spec.ts:86]: typo: Routes

File: src/Lifi.unit.spec.ts [hunk 6]
```diff
@@ -93,24 +94,68 @@ describe('LIFI SDK', () => {
         })
 
         await expect(Lifi.getRoutes(request)).rejects.toThrow(
-          'SDK Validation: Invalid Routs Request'
+          'Invalid Routs Request'
```
Comment (Addminus) [inline src/Lifi.unit.spec.ts:97]: typo: Routes

**Review by Addminus — CHANGES_REQUESTED** [review]
There are multiple typos in test files, all while expecting the same error. That means the typo is somewhere in the codebase

**Review by maxklenk — APPROVED** [review]

**Pull request merged**